soc/v2/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -0,0 +1,60 @@
+# ST Microelectronics STM32 all MCU lines
+
+# Copyright (c) 2017, I-SENSE group of ICCS
+# SPDX-License-Identifier: Apache-2.0
+
+# Here are set all the Kconfig symbols common to the whole STM32 family
+
+if SOC_FAMILY_STM32
+
+config CORTEX_M_SYSTICK
+	default n if STM32_LPTIM_TIMER
```

Comment:
This file looks to have creeped back in again (duplicate)

Reply:
Seen it as well. This is fixed now.